extras/agc-minimal-permissions/lib/policy-statements.ts

```diff
@@ -28,27 +28,11 @@ export class AgcPermissions {
                 
                 effect: Effect.ALLOW,
                 actions: actions(svc,
-                    "CreateVpc",
-                    "DeleteVpc",
-                    "CreateNatGateway",
-                    "DeleteNatGateway",
-                    "CreateSecurityGroup",
-                    "DeleteSecurityGroup",
-                    "CreateInternetGateway",
-                    "DeleteInternetGateway",
-                    "CreateRoute",
-                    "DeleteRoute",
-                    "CreateSubnet",
-                    "DeleteRouteTable",
-                    "AuthorizeSecurityGroupIngress",
-                    "AuthorizeSecurityGroupEgress",
-                    "RevokeSecurityGroupIngress",
-                    "RevokeSecurityGroupEgress",
-                    "CreateVpcEndpoint",
-                    "DeleteVpcEndpoints",
-                    "AllocateAddress",
-                    "AssociateAddress",
-                    "ReleaseAddress",
+                    "Create*",
+                    "Delete*",
+                    "Authorize*",
+                    "Revoke*",
+                    "*Address",
```

Comment:
Policy api threw an error as we are exceeding the limit of characters in policy document and was recommended to use wildcards as I am adding more permissions here for the purpose of (cdk bootstrap).
* Same applies to other section of the changes

Reply:
This breaks the whole idea of "Minimal Permissions". We can always split up the policies into two or more policies.

Reply:
Done